command/backup_start_unmanaged.go

```diff
@@ -33,8 +33,8 @@ To run any other type of backup, see backupshq job run --help.
 
 		backupID := c.Args().Get(0)
 		backup := client.GetBackup(backupID)
-		if backup.Type != api.BACKUP_TYPE_UNMANAGED {
```

Comment:
I noticed a few more things that I think can be removed, again can't comment on unchanged code but it's sort of related to this line here so it'll do :nerd_face: 

The `ListBackup()` function in `api/backup.go` still takes a `backupType` constant as an argument, which isn't used in the function anymore. There's also still a call to it [here](https://github.com/backupshq/agent/blob/master/agent/agent.go#L50) which passes in the type argument. I guess the constants can also be removed?

Reply:
Good catch, at least someone is paying attention! Updated now, thanks.